vspace/src/main/java/edu/asu/diging/vspace/core/references/impl/CoinSMetadataProvider.java

```diff
@@ -0,0 +1,246 @@
+        referenceMap.put(CoinSConstants.ART_TYPE, CoinSConstants.BLOGPOST_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.PATENT_TYPE, CoinSConstants.PATENT_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.PODCAST, CoinSConstants.BLOGPOST_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.SOFTWARE_DOCUMENTATION, CoinSConstants.BLOGPOST_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.BOOK_GENRE, CoinSConstants.BOOK_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.BOOK_SECTION_GENRE, CoinSConstants.BOOK_RFT_VAL_FMT);
+        referenceMap.put(CoinSConstants.CONFERENCE_PAPER, CoinSConstants.BOOK_RFT_VAL_FMT);
+    }
+
+    @Override
+    public ReferenceMetadataType getReferenceMetadataType() {
+        return ReferenceMetadataType.DEFAULT;
+    }
+
+    private String getReferenceData(String referenceType) {
+        return referenceMap.getOrDefault(referenceType, CoinSConstants.DEFAULT_RFT_VAL_FMT);
+    }
+
+    @Override
+    public StringBuffer getReferenceMetadata(Reference reference) throws ReferenceMetadataEncodingException {
+
+        String refTitleEncoded = getRefTitleEncoded(reference.getTitle(), reference.getType());
+        String refAuthorEncoded = getRefAuthorEncoded(reference.getAuthor()) + getRefYearEncoded(reference.getYear());
+        String refJournalEncoded = getRefJournalEncoded(reference.getJournal()) + getRefUrlEncoded(reference.getUrl());
+        String refVolumeEncoded = getRefVolumeEncoded(reference.getVolume()) + getRefIssueEncoded(reference.getIssue());
+        String refPagesEncoded = getRefPagesEncoded(reference.getPages())
+                + getRefEditorsEncoded(reference.getEditors());
+        String refNoteEncoded = getRefNoteEncoded(reference.getNote()) + getRefTypeEncoded(reference.getType());
+        String referenceTypeRFTValueString = getReferenceData(reference.getType());
+        StringBuffer urlEncodedReferenceMetaData = new StringBuffer(CoinSConstants.DEFAULT_URL_VERSION);
+        try {
+            urlEncodedReferenceMetaData.append("&");
+            urlEncodedReferenceMetaData.append(CoinSConstants.DEFAULT_CTX_VERSION);
+            urlEncodedReferenceMetaData.append(CoinSConstants.RFR_ID_TAG);
+            urlEncodedReferenceMetaData
+                    .append(URLEncoder.encode(CoinSConstants.DEFAULT_RFR_ID, StandardCharsets.UTF_8.name()));
+            urlEncodedReferenceMetaData.append(CoinSConstants.RFT_VAL_FMT_TAG);
+            urlEncodedReferenceMetaData.append(referenceTypeRFTValueString);
+            urlEncodedReferenceMetaData.append(refTitleEncoded);
+            urlEncodedReferenceMetaData.append(refAuthorEncoded);
+            urlEncodedReferenceMetaData.append(refJournalEncoded);
+            urlEncodedReferenceMetaData.append(refVolumeEncoded);
+            urlEncodedReferenceMetaData.append(refPagesEncoded);
+            urlEncodedReferenceMetaData.append(refNoteEncoded);
+
+        } catch (UnsupportedEncodingException e) {
```

Comment:
is this thrown in each line or just line 73? If it's just line 73, the try/catch block should be around that line only.